cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/applicationContext.xml

```diff
@@ -84,5 +84,7 @@
             </props>
         </property>
     </bean>
-
+    
+    <bean id="ticketTransactionManager" 
+          class="org.springframework.integration.transaction.PseudoTransactionManager" />
```

Comment:
Are we sure it's a good idea to bring the implementation from spring integration project here? Seems like this impl has been designed for different purposes within the integration and messaging domains...

http://docs.spring.io/spring-integration/api/org/springframework/integration/transaction/PseudoTransactionManager.html